tests/ui/impl-trait/impl-fn-predefined-lifetimes.edition2024.stderr

```diff
@@ -0,0 +1,11 @@
+error[E0792]: expected generic lifetime parameter, found `'_`
+  --> $DIR/impl-fn-predefined-lifetimes.rs:8:9
+   |
+LL | fn a<'a>() -> impl Fn(&'a u8) -> (impl Debug + '_) {
+   |      -- this generic parameter must be used with a generic lifetime parameter
```

Comment:
The last edition changes the span being pointed at in such a way that the wording should change (or get the span from previous editions back).